src/renderer/store/index.js

```diff
@@ -1,11 +1,279 @@
+        commit('setUnconfirmedBalance', data.unconfirmed_balance);   
+        commit('setAvailableBalance', data.balance - data.unconfirmed_balance);   
+      });
+
+
+      client.getInfo(function(err, data, resHeaders) {
+        if (err) {
+          commit('setPeerCount', 'None');
+          commit('setBlockheight', 'Scanning');
+          return console.log(err);
+        } 
+        
+        commit('setPeerCount', data.connections);
+        commit('setBlockheight', data.blocks);
+      }); 
+
+    },
+
+    refreshTransactions({ commit }) {
+      console.log("refreshing transactions");
+
+      var client = new bitcoin.Client({
+        port: this.state.rpcCredentials.port,
+        user: this.state.rpcCredentials.user,
+        pass: this.state.rpcCredentials.password,
+        timeout: 60000
+      });
+
+      client.listTransactions(function(err, data, resHeaders) {
+        if (err) return console.log(err);
+        commit('setTransactions', data)
+      });
+    },      
+
+    addTAddress({ commit }) {
+      var client = new bitcoin.Client({
+        port: this.state.rpcCredentials.port,
+        user: this.state.rpcCredentials.user,
+        pass: this.state.rpcCredentials.password,
+        timeout: 60000
+      });
+
+      // Get T-Addresses
+      client.getNewAddress('', function(err, data, resHeaders) {
+        if (err) return console.log(err);      
+        console.log(data);   
+        commit('addAddress', {address: data, balance: 0, type: 't'});
+      });
+    },
+
+    addZAddress({ commit }) {
+      var client = new bitcoin.Client({
+        port: this.state.rpcCredentials.port,
+        user: this.state.rpcCredentials.user,
+        pass: this.state.rpcCredentials.password,
+        timeout: 60000
+      });
+
+      // Get T-Addresses
+      client.cmd('z_getnewaddress', function(err, data, resHeaders) {
+        if (err) return console.log(err);      
+        console.log(data);   
+        commit('addAddress', {address: data, balance: 0, type: 'z'});
+      });
+    },
+    sendToMany({ commit },transactionForm) {
+      var self = this;
+      var client = new bitcoin.Client({
+        port: this.state.rpcCredentials.port,
+        user: this.state.rpcCredentials.user,
+        pass: this.state.rpcCredentials.password,
+        timeout: 60000
+      });
+     
+      var receivers = [];         
+      for(let receiver of transactionForm.destinationAddresses) {
+        receivers.push({"address":receiver.toString(), "amount": transactionForm.amount});
+      }       
+
+      client.cmd( 'z_sendmany',transactionForm.from,receivers,1,transactionForm.fee , function(err, data, resHeaders) {
+        if (err) return console.log(err);              
+        var operationids =[data.toString()];
+        client.cmd( 'z_getoperationstatus', operationids , function(err, opData, resHeaders) {
+          console.log(opData);
+          if (err || opData[0].status == "failed") {
+            console.log()
```

Comment:
Will this console.log() mess with some browsers?

Reply:
had no issues on chromium, but we can jsut remove it